.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        entry:
+          - { ruby: "2.7" }
+          - { ruby: "2.7", concurrency: async-websocket }
```

Comment:
This concurrency thing is a copy-paste from slack-ruby-client, we don't need it here (or below).